src/sentry/api/endpoints/organization_onboarding_tasks.py

```diff
@@ -26,11 +33,16 @@ def post(self, request, organization):
         ):
             return Response(status=422)
 
+        values = {}
+
+        if status:
+            values["status"] = status
+            values["date_completed"] = timezone.now()
+        if completion_seen:
+            values["completion_seen"] = timezone.now()
+
         rows_affected, created = OrganizationOnboardingTask.objects.create_or_update(
-            organization=organization,
-            user=request.user,
```

Comment:
Was it a bug that we had user here before?

Reply:
Yeah I think so.